extensions/railway/package.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "railway",
+  "title": "Railway",
+  "description": "Search and navigate to your Railway projects",
+  "icon": "command-icon@light.png",
+  "author": "coffee-cup",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "railway-projects",
+      "title": "Railway Projects",
```

We follow the <Verb> <Noun> pattern for command names. 
```suggestion
      "title": "Search Projects",
```